### DISCUSSION TITLE
Why can't anyone make something like this for windows?

### DISCUSSION THREAD

**Discussion by pragnesh-singh-rajput** [zed-industries/zed#8903] [Feature Requests]
It is open source so anyone can make a windows version write ? then why don't someone steps up for this task ?

**Comment by SomeoneToIgnore** [comment]
There are actually people who do something on that front, it's pretty easy to check by using the GitHub search on pull requests:
https://github.com/zed-industries/zed/pulls?q=is%3Apr+windows

**Comment by Moshyfawn** [comment]
Check out [the dedicated Windows port Discord channel](https://discord.com/channels/869392257814519848/1208481909676576818), where you will be amazed at the number of people dedicated to porting Zed to Windows.
